hadoop-ozone/dist/src/main/compose/ozone/docker-compose.yaml

```diff
@@ -18,6 +18,7 @@ version: "3"
 services:
    datanode:
       image: apache/hadoop-runner
+      privileged: true #required by the profiler
```

Comment:
This can be avoided with initContainer running in privileged mode that updates the following
```
            sudo bash -c 'echo 1 > /proc/sys/kernel/perf_event_paranoid'
            sudo bash -c 'echo 0 > /proc/sys/kernel/kptr_restrict'
```
with this initContainer will apply the required changes and will complete. main container can still run in non-previleged mode.

Reply:
Sorry, I missed the part that it is running using docker-compose and not k8s.

Reply:
Thanks the comment. I think we can assume that the kernel parameters are adjusted. I will test it without the privileged flag and remove those lines..